src/Our.Umbraco.Ditto/ComponentModel/ValueResolvers/AppSettingValueResolver.cs

```diff
@@ -28,12 +28,21 @@ public class AppSettingValueResolver : DittoValueResolver<AppSettingAttribute>
         /// </returns>
         public override object ResolveValue(ITypeDescriptorContext context, AppSettingAttribute attribute, CultureInfo culture)
         {
-            if (string.IsNullOrWhiteSpace(attribute.AppSettingKey))
+            // default to attribute value
+            string appSettingKey = attribute.AppSettingKey;
```

Comment:
happy to update it - is it safe to assume context.PropertyDescriptor.Name will never be null ?